assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,33 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
```

Comment:
does it work with the lazyload experiment?

Reply:
Yes